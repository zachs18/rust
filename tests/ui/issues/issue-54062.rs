use std::sync::Mutex;

struct Test {
    comps: Mutex<String>,
}

fn main() {}

fn testing(test: Test) {
    let _ = test.comps.state.inner.try_lock();
    //~^ ERROR: field `state` of struct `Mutex` is private
}
