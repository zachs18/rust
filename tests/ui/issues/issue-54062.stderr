error[E0616]: field `state` of struct `Mutex` is private
  --> $DIR/issue-54062.rs:10:24
   |
LL |     let _ = test.comps.state.inner.try_lock();
   |                        ^^^^^ private field

error: aborting due to previous error

For more information about this error, try `rustc --explain E0616`.
